webapp/templates/team/menu.html.twig

```diff
@@ -54,7 +54,15 @@
             {% endif %}
         </ul>
 
-        <div class="ms-auto me-3">
+        <div class="ms-auto me-3 d-flex gap-3">
+            {% if team is defined and team.unreadClarifications|length > 0 %}
+                <a class="nav-link" href="{{ path('team_clarification', {clarId: team.unreadClarifications[0].clarid}) }}">
```

Comment:
proiterujes je a udelas dropdown s nekolika odkazy (na jednotlive messages)?
Kdyz jim prijde 5 zprav, tak at si je mohou postupne vsechny otevrit.

Zaroven bych pridal do badge pocet novych zprav
![image](https://github.com/delta-cs/seminar-domjudge/assets/20317551/46db9eef-87b2-4961-8d85-33db9c9ecd71)

Reply:
maji se klinutim v dropdownu otevirat na nove karte? @zapotocnylubos

Reply:
asi ne, to by rozbilo ten UX rekl bych

Reply:
zkontroluj ale prosim, jestli se po prokliknuti (a tedy nacteni cele stranky) aktualizuje pocet tech neprectenych zprav. Aby to proste nebylo o krok pozadu, jestli to dava logiku

Reply:
jojo, proklik je v pohode. pri prechodu zpatky (sipka zpet v prohlizeci) to teda nacte stejnou stranku, tedy neupdatne se to, ale to je asi obvykle.

Reply:
to je v pohode